.github/renovate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Runs Renovate with solrbot, see dev-docs/dependency-upgrades.adoc for more",
+  "enabled": true,
+  "dependencyDashboard": false,
+  "enabledManagers": ["gradle"],
+  "includePaths": ["versions.*", "build.gradle"],
+  "postUpgradeTasks": {
```

Comment:
This is great, and a step that's missing in the dependabot feature for the solr-operator.